modules/oxide/src/less/theme/components/view/view.less

```diff
@@ -1,4 +1,4 @@
-@view-header-padding: @pad-sm @pad-sm 0 @pad-sm;
+@view-header-padding: (@pad-sm + 2) (@pad-sm + 2) 2px (@pad-sm + 2);    // Add 2px to account for outline
```

Comment:
2 is great until we change the outline 😂 

but I don't know if the outline width is a variable.

Reply:
There are two variables that could be used: `@keyboard-focus-outline-width` and `@tinymce-border-width`. Decide to go with the first one because it is also used in style for focus outline on the buttons